packages/esm-home-app/src/index.ts

```diff
@@ -24,6 +25,8 @@ export const homeWidgetDbLink = getSyncLifecycle(createDashboardLink(dashboardMe
 
 export const homeWidgetDashboard = getSyncLifecycle(homeWidgetDashboardComponent, options);
 
+export const homeMetricsWidgetDashboard = getSyncLifecycle(HomeMetricsWidgetsComponent, options);
```

Comment:
Please use `getAsyncLifecycle()`. Basically, we should _always_ use `getAsyncLifecycle()` as it improves app performance.

Reply:
Noted, thanks.